libraries/ms-drivers/src/ltc_afe.c

```diff
@@ -0,0 +1,43 @@
+#include "ltc_afe.h"
+#include "critical_section.h"
+#include "ltc_afe_fsm.h"
+#include "ltc_afe_impl.h"
+
+StatusCode ltc_afe_init(LtcAfeStorage *afe, const LtcAfeSettings *settings) {
+  status_ok_or_return(ltc_afe_impl_init(afe, settings));
+  return ltc_afe_fsm_init(&afe->fsm, afe);
+}
+
+StatusCode ltc_afe_set_result_cbs(LtcAfeStorage *afe, LtcAfeResultCallback cell_result_cb,
+                                  LtcAfeResultCallback aux_result_cb, void *context) {
+  LtcAfeSettings *settings = &afe->settings;
+  if (settings->cell_result_cb != NULL || settings->aux_result_cb != NULL) {
+    return status_code(STATUS_CODE_INVALID_ARGS);
+  }
+
+  bool disabled = critical_section_start();
```

Comment:
I don't think the necessity has changed, although I guess it might be worth limiting users to set callbacks only during init to remove the need for this method?

Reply:
eh I guess that's probably fine to just note for implementation of cell sense, seems fine then